### DISCUSSION TITLE
Add constants for minimum and maximum values for int and float data types

### DISCUSSION THREAD

**Discussion by blackears** [godotengine/godot-proposals#6575] [Scripting]
It can sometimes be useful to set a variable to the largest or smallest possible value.  I'd like to see the following constants added to GDScript:

INT_MAX
INT_MIN
FLOAT_MAX
FLOAT_MIN

**Comment by AThousandShips** [comment]
Already tracked by #2411

**Comment by Calinou** [comment]
This is already tracked by https://github.com/godotengine/godot-proposals/issues/2411.
